spec/import/mods_transform_spec.rb

```diff
@@ -4,7 +4,13 @@
 
 RSpec.describe 'Transforming MODS files' do
   let(:provide) do
-    { 'identifier' => identifier, 'exhibit_slug' => slug, 'inst_id' => 'stanford' }
+    {
```

Comment:
You're right; I've switched things around a little bit now so it does get used. Keeping it around will be useful when we add additional MODS mappings.